src/docker-runner/utils.ts

```diff
@@ -0,0 +1,68 @@
+import execa from 'execa';
+import shell from 'shelljs';
+import yargs from 'yargs';
+import { hideBin } from 'yargs/helpers';
+// Import { log } from '../log';
+
+export const isDockerInstalled = () => {
+  if (!shell.which('docker')) {
+    throw new Error('Docker is not installed');
+  }
+};
+
+const isLostPixelImageDownloaded = async ({ version }: { version: string }) => {
+  isDockerInstalled();
+  const { exitCode, stdout, stderr } = await execa('docker', [
+    'images',
+    '-q',
+    `lostpixel/lost-pixel:${version}`,
+  ]);
+
+  if (exitCode !== 0) {
+    throw new Error(`Not successful docker operation, ${stderr}`);
+  }
+
+  return stdout.trim().length > 0;
+};
+
+export const downloadImageIfNotExistent = async ({
+  version,
+}: {
+  version: string;
+}) => {
+  const doesImageExist = await isLostPixelImageDownloaded({
+    version,
+  });
+  if (!doesImageExist) {
+    await execa('docker', ['pull', `lostpixel/lost-pixel:v${version}`]);
+  }
+};
+
+type ParsedYargs = {
+  configDir: 'string';
+};
+
+export const executeDockerRun = async ({ version }: { version: string }) => {
+  await downloadImageIfNotExistent({ version });
+
+  // @ts-expect-error TBD
+  const argv = yargs(hideBin(process.argv)).parse() as ParsedYargs;
+
+  const args = [
+    'run',
+    '--rm',
+    '-v',
+    '$PWD:/workspace',
```

Comment:
Will this be resolved correctly, or do we need to provide the actual working dir?

Reply:
I wrote you on slack alreayd, it's quite surprising. I think the volume is copied over correctly and config is picked up - when the process starts collecting the stories to shoot it fails for some reason 😢